File: app/src/main/java/io/eodc/planit/activity/SettingsActivity.java
package io.eodc.planit.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

import butterknife.BindView;
import butterknife.ButterKnife;
import io.eodc.planit.R;

/**
 * Activity for the settings page
 *
 * @see io.eodc.planit.fragment.SettingsFragment
 */
public class SettingsActivity extends AppCompatActivity {
    @BindView(R.id.toolbar) Toolbar mToolbar;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
        ButterKnife.bind(this);

        setSupportActionBar(mToolbar);
        if (getSupportActionBar() != null) getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }
}

File: app/src/main/java/io/eodc/planit/activity/SettingsActivity.kt
package io.eodc.planit.activity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity

import io.eodc.planit.R
import kotlinx.android.synthetic.main.activity_settings.*

/**
 * Activity for the settings page
 *
 * @see io.eodc.planit.fragment.SettingsFragment
 */
class SettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)

        setSupportActionBar(tb)
        if (supportActionBar != null) supportActionBar!!.setDisplayHomeAsUpEnabled(true)
    }
}
